_sass/main.scss

```
@@ -227,56 +262,220 @@ div#presets{
   padding: 10px;
 }
 
+.card__law {
+  padding: 16px 27px;
+
+  &~& {
+    border-top: 1px solid $color-gray-light;
+  }
+
+  &-header {
+    font-size: 16px;
+    line-height: 21px;
+    display: flex;
+    gap: 24px;
+    margin-bottom: 8px;
+  }
+
+  &-header-title {
+    font-weight: 700;
+  }
+
+  &-header-status {
+    font-size: 14px;
+    color: $key-color;
+  }
+
+  &-content {
+    margin-bottom: 12px;
+  }
+}
+
+.accordion {
+  background-color: $color-white;
+  border-top: 1px solid $color-gray-light;
+  border-bottom: 1px solid $color-gray-light;
+
+  &__title {
+    align-items: center;
+    display: flex;
+    height: 42px;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 21px;
+    padding: 0 27px;
+    cursor: pointer;
+    color: $color-black-text;
+
+    &:before {
+      content: '';
+      display: inline-flex;
+      width: 12px;
+      height: 12px;
+      margin-right: 12px;
+      background-image: url(/assets/images/arrow.svg);
+      background-repeat: no-repeat;
+      background-position: center;
+      background-size: contain;
+    }
+  }
+
+  &_expand {
+    .accordion__title:before {
+      transform: rotate(180deg);
+    }
+  }
+
+  &__content {
+    font-size: 16px;
+    line-height: 21px;
+    border-top: 1px solid $color-gray-light;
+    display: none;
+    color: $color-black-text;
+
+    &:not(&_wide) {
+      padding: 16px 27px;
+    }
+  }
+
+  &~& {
+    border-top: none;
+  }
+}
+
 div#card {
   font-family: Arial;
   font-size: $font-size-m;
   line-height: 21px;
   top: 0;
-  right: -18px;
   left: auto;
-  width: 506px;
+  width: 488px;
   height: 100%;
   background:#F4F4F4;
-  padding: 32px 32px 32px 68px;
+  padding-left: 40px;
   overflow: auto;
```

скролл у карточки горизонтальный на мобильном устройстве (1334 x 750)
```suggestion
  overflow-x: hidden;
  overflow-y: auto;
```